resources/src/jsMain/kotlin/dev/icerock/moko/resources/LocalizedStringLoader.kt

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright 2022 IceRock MAG Inc. Use of this source code is governed by the Apache 2.0 license.
+ */
+
+package dev.icerock.moko.resources
+
+import kotlinx.browser.window
+import kotlinx.coroutines.await
+import kotlin.js.Json
+
+class LocalizedStringLoader {
+    private val cachedLocalizationFiles: MutableMap<SupportedLocale, Json> = mutableMapOf()
+
+    private var cachedFallbackFile: Json? = null
+
+    suspend fun loadLocalizedString(
```

Comment:
we should find way without suspend at every `localized` calls. i think some preload logic required.

Reply:
I feel like it's cleaner that way. In my personal project using compose web this simple composable does a nice job:
`@Composable
fun stringResource(resource: StringResource): String {
    var value by remember { mutableStateOf("") }

    LaunchedEffect(resource) {
        value = resource.localized()
    }

    return value
}
`